src/main/java/ro/vladutit/Don/t/forget/v2/repository/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package ro.vladutit.Don.t.forget.v2.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import ro.vladutit.Don.t.forget.v2.model.User;
+
+import java.util.Optional;
+
+public interface UserRepository extends CrudRepository<User, Long> {
```

Comment:
why is this empty?

Reply:
Because at the moment I use in Service only the save method that I have in the CRUD class which is inherited from the Repository class. I don't have to write it here again. In the future, a findByUsername and a findByEmail will appear, but I didn't write them, because then you probably told me why I wrote them, because I still didn't use them. I will definitely use them for validations to check their uniqueness.